edge-modules/edge-dashboard/src/modules.rs

```diff
@@ -0,0 +1,35 @@
+use url::Url;
+use edgelet_http_mgmt::*;
+use edgelet_core::{Module, ModuleRuntime};
+use futures::future::Future;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct SparseModule {
+    name: String,
+}
+
+pub fn get_management_uri(contents: &str) -> Option<String> {
```

Comment:
As commented earlier, we should try to use the Yaml parsing code from `iotedged`.